tests/rest/admin/test_event_reports.py

```diff
@@ -410,6 +410,35 @@ def _check_fields(self, content: List[JsonDict]) -> None:
             self.assertIn("score", c)
             self.assertIn("reason", c)
 
+    def test_count_correct_despite_table_deletions(self) -> None:
+        """
+        Tests that the issue fixed by https://github.com/matrix-org/synapse/pull/13525
+        does not recur:
+        the count should match the number of rows, even if rows in joined tables
+        are missing.
```

Comment:
```suggestion
        Tests that the count matches the number of rows, even if rows in joined tables
        are missing.
```

Not sure it's worth mentioning the PR in the docstring. I'm not feeling incredibly strongly about it though so happy to keep it if you disagree.